packages/blitz-rpc/src/loader-server.ts

```diff
@@ -90,18 +93,19 @@ export async function recursiveFindResolvers(
       const pathStat = await promises.stat(absolutePath)
 
       if (pathStat.isDirectory()) {
-        await recursiveFindResolvers(absolutePath, filter, ignore, arr, rootDir)
+        await recursiveFindResolvers(absolutePath, rpcFolders, filter, ignore, arr, rootDir)
         return
       }
 
       if (!filter.test(part)) {
         return
       }
 
-      const relativeFromRoot = absolutePath.replace(rootDir, "")
-      if (getIsRpcFile(relativeFromRoot)) {
-        arr.push(relativeFromRoot)
-        return
+      if (getIsRpcFile(absolutePath)) {
+        if (!absolutePath.includes("@blitzjs")) {
```

Comment:
shouldn't need blitzjs ignore here

Reply:
Due to the inclusion of the other "rpc" folders.  If I remove this line then everything breaks.  Other files in the root, do not have this issue.

absolutePath example:

> /Users/paul/development/src/github/bar2/blitz/packages/canvas/node_modules/@blitzjs/auth/node_modules/blitz/node_modules/@blitzjs/generator/templates/app/app/auth/mutations/forgotPassword.ts

Unless there already exists a function to clean this up?

Reply:
Ah, then further up in this function we should filter out `node_modules` so we never search inside of it.

Reply:
Ok I have done the following:

```
if (absolutePath.includes("@blitzjs")) {  
    return
}

if (getIsRpcFile(absolutePath)) {                
    arr.push(absolutePath)
    return        
}
```